### PR TITLE
Added resources to the readme

### PR DESCRIPTION
**Pull request by ruiined**
## Before you start

1. Have you run `git pull origin main` on your branch to bring in the latest changes?
2. Is there an issue on the repository for this request? If not, please create one.
3. Have you followed the correct workflow in the [team documentation](https://docs.google.com/document/d/13yi709ax4nmfTG9J0F57yEFpYRH7fdtczwvVJCrNIsM/edit#)?
4. Conflict resolution to be dealt with **on GitHub**.

## Checklist before requesting a review

- [ ] I have performed a self-review of my code
- [ ] I have added thorough tests.
- [ ] I have created a PR ticket on the [Trello board](https://trello.com/b/P8lEsvZe/acebook).
- [ ] I will link the PR to the Trello card once created.
- [ ] I will send the PR in the Slack channel and ping @here.

### CODE REVIEW

**Review by dennihous — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -25,9 +25,8 @@ Travel Angel is a MERN-stack web application used to safely and securely store t
 
 - [Mongoose](https://mongoosejs.com) - Object Data Mapper for MongoDB.
 - [Nodemon](https://nodemon.io/) - reloads the server automatically on file-change (Not CSS though!).
-- [ESLint](https://eslint.org) - project-wide linting.
-- [Jest](https://jestjs.io/) - unit tests.
-- [Cypress](https://www.cypress.io/) - integration tests.
+- [Jest](https://jestjs.io/) - back-end unit tests.
+- [React Testing Library](https://www.cypress.io/) - front-end unit tests.
```
Comment (heykathl) [inline README.md:29]: Wrong link? It's linking to cypress

**Review by heykathl — CHANGES_REQUESTED** [review]
Link needs changing? Thanks for fixing the readme Maria! 😊

File: README.md (hunk 1 above)
Comment (ruiined) [reply README.md:29]: oops

Comment (ruiined) [reply README.md:29]: good spot!

**Review by dennihous — APPROVED** [review]

**Review by heykathl — APPROVED** [review]
Amazing! Thanks Maria 🤩

**Pull request merged**